### PR TITLE
changed QuantConnect.IBAutomater.csproj to SDK style

### PR DESCRIPTION
**Pull request by vslee**
- it was previously MSBuild style
- this changes it to the newer SDK style, which will allow further options in the future
- SDK style is more concise: a lot of settings which were explicitly specified before are now the defaults
   - App.Config, packages.config, and AssemblyInfo.cs were able to be removed as their information is now in QuantConnect.IBAutomater.csproj
- functional settings have been preserved, including targeting .net framework 4.5.2
   - versions of dependencies such as Newtonsoft.Json 10.0.3 and NodaTime 1.3.4 have been preserved
   - NuGet settings have not been touched
- CSharpDemoIBAutomater remains as a MSBuild style project targeting .net framework 4.5.2 to show that this does not break current projects
- part of #15

### CODE REVIEW

**Comment by vslee** [comment]
rebased to master, solving merge conflicts

**Comment by jaredbroad** [comment]
HI @vslee! Welcome! Thanks for the PR we'll review and try and get it merged today.

File: QuantConnect.IBAutomater/QuantConnect.IBAutomater.csproj [hunk 1]
```diff
@@ -1,72 +1,22 @@
...
   </ItemGroup>
-  <ItemGroup />
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Include="NodaTime" Version="1.3.4" />
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Properties\" />
+  </ItemGroup>
```
Comment (StefanoRaggi) [inline QuantConnect.IBAutomater/QuantConnect.IBAutomater.csproj]: I think we can safely remove these lines

**Review by StefanoRaggi — CHANGES_REQUESTED** [review]
Thank you for the PR @vslee
I just tested the changes and pushed v1.0.30 to NuGet

Just a few minor changes requested:
- remove the `Properties` folder from the project file
- update the demo project to reference the latest NuGet package (v1.0.30)
- update `QuantConnect.IBAutomater.nuspec` for the new build output path:
these are the updated lines:
```
    <version>1.0.30</version>
    ...
    <file src="net452\QuantConnect.IBAutomater.exe" target="lib\net45" />
    <file src="net452\QuantConnect.IBAutomater.pdb" target="lib\net45" />
    ...
    <file src="net452\IBAutomater.bat" target="build" />
    <file src="net452\IBAutomater.sh" target="build" />
```

**Review by StefanoRaggi — APPROVED** [review]
Thank you for the quick fixes 👍

**Pull request merged**